lighthub/inputs.cpp

```diff
@@ -302,62 +309,32 @@ void Input::onContactChanged(int val)
   }
 }
 
-void Input::onEncoderChanged0() {
```

Comment:
Does you mean **counter** here? (not encoder)
Rotary encoder I'm also have in roadmap, but that is different
What about rename?

Reply:
You are right! Counter as long as direction does not matter.

Reply:
What about renaming?